webapp/src/App.tsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
-import { Route, Routes } from 'react-router-dom';
 import './App.css';
+import MainScreen from './components/organisms/MainScreen/MainScreen';
+import { Auth0Provider } from '@auth0/auth0-react';
+import { Route, Routes } from 'react-router-dom';
```

Comment:
I also do a lot of re-ordering with imports. Just keep in mind it may cause merge issues